### PR TITLE
Re-enable PyLint

### PR DESCRIPTION
**Pull request by VijaySharma815**
Pylint list of errors:
https://vald-phoenix.github.io/pylint-errors/#list-of-errors

### CODE REVIEW

File: cosmpy/crypto/address.py [hunk 1]
```diff
@@ -94,6 +94,7 @@ def __bytes__(self):
 
     @property
     def data(self):  # noqa:
+        """Return address details"""
```
Comment (5A11) [inline cosmpy/crypto/address.py:97]:
```suggestion
        """Return address in string."""
```

File: tests/integration/test_tx.py [hunk 2]
```diff
@@ -31,19 +31,24 @@
...
 
     def get_wallet_1(self):
+        """Get wallet"""
         faucet_api = FaucetApi(NetworkConfig.fetchai_stable_testnet())
         wallet1 = LocalWallet.generate()
         faucet_api.get_wealth(wallet1.address())
         return wallet1
 
     def get_wallet_2(self):
+        """Get wallet"""
```
Comment (5A11) [inline tests/integration/test_tx.py:51]:
```suggestion
        """Get wallet 2."""
```

File: tox.ini [hunk 3]
```diff
@@ -99,6 +99,13 @@ skipsdist = True
 skip_install = True
 deps =
   pylint==2.14.5
+  requests==2.28.1
+  ecdsa==0.18.0
+  pytest==7.1.2
+  grpcio==1.47.0
+  bip-utils==2.6.0
+  blspy==1.0.14
+  bech32==1.2.0
```
Comment (5A11) [inline tox.ini:108]: Why are these added?

File: .pylintrc [hunk 4]
```diff
@@ -1,17 +1,30 @@
...
+# See here for more options: https://www.codeac.io/documentation/pylint-configuration.html
+## Eventually resolve these:
+# W0707: raise-missing-from
+
+## Eventually decide on a logging policy:
+# W1202: logging-format-interpolation
+# W1203: logging-fstring-interpolation
+
+## Keep the following:
+# C0103: invalid-name, # kept as no harm
```
Comment (5A11) [inline .pylintrc:13]:
```suggestion
# C0103: invalid-name, # REST method names are equivalent to their gRPC counterparts and those are autogenerated with invalid names according to PyLint
```

File: tox.ini (hunk 3 above)
Comment (5A11) [reply tox.ini:108]: Needed for PyLint to work and recognise the imports.

**Comment by 5A11** [comment]
Please update the list of ignored errors according to what we discussed before and address the complaints raised by the ones we are not ignoring.

File: cosmpy/aerial/contract/__init__.py [hunk 5]
```diff
@@ -70,6 +70,7 @@ def __init__(
         :param address: address, defaults to None
         :param digest: digest, defaults to None
         """
+        super().__init__(address)
```
Comment (VijaySharma815) [inline cosmpy/aerial/contract/__init__.py:73]: @5A11 Pleaser verify.

File: cosmpy/crypto/address.py [hunk 6]
```diff
@@ -51,6 +51,7 @@ def __init__(
         :param value: str, byte, public key or Address another instance
         :param prefix: optional string
         """
+        super().__init__(value)
```
Comment (VijaySharma815) [inline cosmpy/crypto/address.py:54]: @5A11 please verify.

**Review by 5A11 — APPROVED** [review]
lgtm
